src/redprl/redprl.grm

```diff
@@ -333,23 +338,28 @@ atomicRawTac
 
   | operator LPAREN bindings RPAREN (Ast.$$ (operator, bindings))
   | metavar metavarArgs (Ast.$$# (metavar, metavarArgs))
-  | IDENT (`` IDENT)
```

Comment:
Sounds good. I think this might have just been a red herring during development: at one point it fixed a bunch of conflicts, but it looks like later changes have obviated the need to remove it. I'll put it back now.